src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider2.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.ObjectModel;
+using System.Reflection;
+
+namespace System.Runtime.Serialization
+{
+    public interface ISerializationSurrogateProvider2 : ISerializationSurrogateProvider
```

Comment:
This is a new public type. Do we need API review for it?

Also, it needs doc comments.

Reply:
We have had two API reviews for this PR. #72242 covered the new API's being exposed by the runtime component of this PR (including this new interface), and #72243 covered the 'new' API presented by the schema importing component of this PR.

Reply:
Sounds good. It would be useful to mention these issues in the PR description.

Reply:
For the types being exposed from System.Private.DataContractSerialization (DataContract, DataContractSet, DataMember, XmlDataContract) - we don't really intend for those types to be consumed by anyone. It was necessary to expose them to allow the schema import work to happen in an external package, but we don't want to encourage anybody else to use them. Is there any guidance about how the doc comments should look for those types?